assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-tiles.scss

```
@@ -0,0 +1,112 @@
+/**
+ * Audience Segmentation Tiles styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-info-tooltip__content.googlesitekit-audience-tiles-info-tooltip__content {
+	color: $c-surfaces-inverse-on-surface;
+
+	a {
+		color: $c-surfaces-inverse-on-surface;
+		text-decoration: underline;
+	}
+}
+
+.googlesitekit-plugin {
+
+	.googlesitekit-widget.googlesitekit-widget__audience-tiles {
+		border-radius: $br-sm;
+
+		.googlesitekit-widget__audience-tiles-tabs {
+			border-bottom: 1px solid $c-utility-divider;
+
+			.mdc-tab-scroller {
+				// Fix ripple escaping the curved corners of the widget.
+				border-top-left-radius: $br-sm;
+				border-top-right-radius: $br-sm;
+				.mdc-tab-scroller__scroll-content {
+					.mdc-tab {
+						flex: none;
+
+						height: 56px;
+
+						// Remove pointer events from the button so that the tooltip inside can be hovered upon
+						pointer-events: none;
+						.mdc-tab__content {
```

Again, please amend for whitespace consistency:
```suggestion
						flex: none;
						height: 56px;

						// Remove pointer events from the button so that the tooltip inside can be hovered upon
						pointer-events: none;

						.mdc-tab__content {
```